plugins/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: plugin
+title: Plugins
+---
+# 1. Plugins
+
+Plugins are tools to extend and modify the functionality of Foreman. The core of Foreman is designed to be lean, to maximize flexibility and minimize code bloat. Plugins offer custom functions and features so that each user can tailor their environment to their specific needs. Foreman plugins are implemented as [rail engines](http://guides.rubyonrails.org/engines.html) that are packaged as [gems](http://guides.rubygems.org/what-is-a-gem/) and thus easily installed into Foreman.
```

Comment:
> to minimize code bloat -- parallel structure

What does this mean?

FWIW, I didn't write any of this, it's just a copy and paste from what's been in the manual forever.